Dockerfile

```diff
@@ -10,4 +12,5 @@ EXPOSE 4443
 
 WORKDIR /go/src/github.com/docker/notary
 
-CMD [ "notary-server", "-config", "cmd/notary-server/dev-config.json" ]
+
+CMD [ "notary-server", "-config", "cmd/notary-server/stage-config.json" ]
```

Comment:
There currently isn't a way to pass runtime environment variables through docker-compose to the Dockerfile. (https://github.com/docker/compose/issues/1377).  I'll talk to infra.

@thaJeztah  '-config' is a byproduct of the command line library used by notary.